build-conventions-secant/src/main/kotlin/secant.android-build-conventions.gradle.kts

```diff
@@ -22,6 +22,9 @@ pluginManager.withPlugin("com.android.application") {
             if (project.property("IS_USE_TEST_ORCHESTRATOR").toString().toBoolean()) {
                 testInstrumentationRunnerArguments["clearPackageData"] = "true"
             }
+            if (project.property("IS_ANDROID_INSTRUMENTATION_TEST_COVERAGE_ENABLED").toString().toBoolean()) {
```

Comment:
On local emulators, shouldn't we be relying on Android Test Services to locate the coverage files?

The only place we use the external storage path explicitly is with emulator.wtf, and that's handled separately in the convention plugin for emulator.wtf (since it doesn't have Android Test Services installed)

Reply:
Yes, agree. Removed from all three android plugins definitions. Thanks for pointing this out.